docs/content/Configuration/Databases/Firebolt.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Firebolt
+permalink: /config/databases/firebolt
+---
+
+## Prerequisites
+
+- The username/password for the [Firebolt][firebolt] account
+
+## Setup
+
+### <--{"id" : "Setup"}--> Manual
+
+Add the following to a `.env` file in your Cube.js project:
+
+```bash
+CUBEJS_DB_NAME=firebolt_database
+CUBEJS_DB_USER=firebolt_user
+CUBEJS_DB_PASS=**********
+CUBEJS_FIREBOLT_ENGINE_ENDPOINT=engine_endpoint
```

Comment:
We have been using `CUBEJS_DB_URL` for other drivers (druid, elasticsearch) Can we use that for consistency?

Reply:
snowflake or clickhouse use `CUBEJS_DB_NAME`
It is more like a name without port or host, not url